src/styles/homeForm.scss

```diff
@@ -10,23 +10,55 @@
         text-align: center;
         background: $baseColor !important;
     }
+
+    &-sider-children {
+        display: flex;
+        align-items: center;
+    }
+}
+
+.ant-table-content {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.ant-space-item {
+    display: flex;
+    align-items: center;
+   
 }
 
 .logo {
-    height: 40px;
+    height: 58px;
+    width: 45px;
+}
+
+.ant-row {
+    flex-flow: nowrap !important;
+}
+
+.ant-input {
+    width: fit-content !important;
+}
+
+.ant-table-pagination-right {
+    justify-content: center !important;
```

Comment:
can we do this without important statements?

Reply:
no, I tried